build/themes/default.html

```diff
@@ -484,7 +485,7 @@
     @media print {
         @page {
             /* suggested printing margin */
-            margin: 0.75in;
+            margin: 1in;
```

Comment:
Oops looks like we changed the margins back to 1 inch from 0.75

Reply:
Sorry, I forgot to make that change in my local master copy of the front-end since we did it quick on your computer. We can change that back today.